server/models/User.py

```diff
@@ -10,31 +11,23 @@ class User(db.Model):
     email = db.Column(db.String(128), nullable=False, unique=True)
     password = db.Column(db.String(128), nullable=False)
     isChef = db.Column(db.Boolean, nullable=False)
-    aboutMe = db.Column(db.String(600))
-    streetAddress = db.Column(db.String(200))
-    city = db.Column(db.String(50))
-    state = db.Column(db.String(50))
-    zipcode = db.Column(db.String(25))
-    country = db.Column(db.String(50))
+
+    address = db.Column(db.Text)
+    generalLocation = db.Column(db.Text)
+
     latitude = db.Column(db.Float)
     longitude = db.Column(db.Float)
-    formattedAddress = db.Column(db.String)
 
-    # Todo: this method of initializing with default values feels very sloppy and a better way to do it probably exists
-    def __init__(self, name, email, password, confirmPassword, aboutMe="", streetAddress="", city="", state="", zipcode="", country="", latitude=0.0, longitude=0.0, formattedAddress=""):
+    aboutMe = db.Column(db.Text)
+    chefDescription = db.Column(db.Text)
```

Comment:
Again, is there a deffereince between aboutMe and chefDescription? I think in theory they are the same thing, we can reuse aboutMe as chefDescription